Ruins/src/main/java/atomicstryker/ruins/common/RuinTemplate.java

```diff
@@ -743,22 +752,41 @@ else if (line.startsWith("preventRotation"))
                 }
                 else if (line.startsWith("adjoining_template"))
                 {
-                    // syntax: adjoining_template=<template>;<relativeX>;<allowedYdifference>;<relativeZ>[;<spawnchance>]
+                    // syntax: adjoining_template=<template>[,<template>[,..]];<relativeX>;<allowedYdifference>;<relativeZ>[;<spawnchance>[;<useRelativeY>;<relativeY>]

```

Comment:
Why the addition of "useRelativeY" at all? Simply use a default value of relativeY=0 to cover the useRelativeY=0 case?